sdk/cosmosdb/cosmos/package.json

```diff
@@ -106,8 +106,8 @@
   },
   "devDependencies": {
     "@azure/dev-tool": "^1.0.0",
-    "@azure/eslint-plugin-azure-sdk": "^3.0.0",
```

Comment:
The change is only for order of dependencies, is it expected?

Reply:
Yes it is require. I added azure identity back to the dev dependencies

Reply:
The change is not for identity package but for 'enlist-plugin-azure-sdk'.

Reply:
ESLint dependency was already present and its the same version as well. Unless the order of packages listed matter it's a no-op and is the question.

Reply:
The effective changes to the file are none, unless the order of listing the package has significance. 
In-case the order has no significance it's better to revert the changes to file.

Reply:
This dependency exists before and also exists after this change, that true. My clarification is why is change to this file needed?

Its only moving it from line 109 to 110, effectively no change (unless ordering is a requirement).
If not let's, please revert the changes to this file.